reconcile/test/ocm/test_utils_ocm_cluster_groups.py

```diff
@@ -0,0 +1,123 @@
+import httpretty as httpretty_module
+
+from reconcile.test.ocm.conftest import (
+    register_ocm_delete_request,
+    register_ocm_get_list_request,
+    register_ocm_post_request,
+)
```

Comment:
This isn't a good pytest practice to import methods from `conftest.py`; please consider the usage of fixtures. See `gql_class_factory` for how to create a fixture that takes arguments.

Reply:
i reworked those functions into fixture functions, following the pattern of `gql_class_factory`